src/sfctl/commands.py

```diff
@@ -203,6 +203,12 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
                 group.command('command', 'invoke_infrastructure_command')
```

Comment:
Add help text for the property group now

Reply:
Is an additional help file required for a standard command group?

Reply:
You added a group of commands: `sfctl property`. In the same way there's documentation on `sfctl application`. Take a look at how the help is defined for that group.

Reply:
Ok will do, thanks.